sg_jira/bridge.py

```diff
@@ -391,3 +398,18 @@ def sync_in_shotgun(
             logger.exception(e)
             raise
         return synced
+
+    @staticmethod
+    def __get_hook_class(hook_path):
+        """"""
+        if not hook_path or not os.path.exists(hook_path):
+            return JiraHook
+        file_name = os.path.splitext(os.path.basename(hook_path))[0]
+        module_name = f"{uuid.uuid4().hex}.{file_name}"
+        spec = importlib.util.spec_from_file_location(module_name, hook_path)
+        module_obj = importlib.util.module_from_spec(spec)
+        sys.modules[module_name] = module_obj
+        spec.loader.exec_module(module_obj)
+        for cls in inspect.getmembers(module_obj, inspect.isclass):
```

Comment:
I'd like to unpack the values in the for to improve readability:

```suggestion
        for name, value in inspect.getmembers(module_obj, inspect.isclass):
```

with self-describing variable names.